### ISSUE TITLE
`onItemClick` parameter is not used and should be removed

### ISSUE THREAD

**Issue by dturner** [android/nowinandroid#614]
The `onItemClick` parameter isn't used. It should be removed. https://github.com/android/nowinandroid/blob/ebfbb5bafd1d13119fa4cdf75f0176ad84cb33eb/core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardList.kt#L41

**Comment by vishesh211** [comment]
@dturner I want to contribute to this issue. I request you to assign it to me.

**Comment by dturner** [comment]
All yours :)

**Comment by chanjungkim** [comment]
Will this be closed?

**Comment by vishesh211** [comment]
Yes it is completed and a PR has been already linked to this issue.

**Comment by gururani-abhishek** [comment]
It's been more than a month, @dturner @vishesh211, can I work on this?

**Comment by gururani-abhishek** [comment]
@dturner please have a look :)
